iree/compiler/Dialect/Util/Transforms/FoldGlobals.cpp

```diff
@@ -356,6 +356,9 @@ static bool deduplicateConstantGlobals(GlobalTable &globalTable) {
 class FoldGlobalsPass
     : public PassWrapper<FoldGlobalsPass, OperationPass<mlir::ModuleOp>> {
  public:
+  explicit FoldGlobalsPass() {}
```

Comment:
`= default;` on these

Reply:
still needed {} for the copy constructor:

> ```[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30037\include\memory(3416): error C2280: 'mlir::iree_compiler::IREE::Util::`anonymous-namespace'::FoldGlobalsPass::FoldGlobalsPass(const mlir::iree_compiler::IREE::Util::`anonymous-namespace'::FoldGlobalsPass &)': attempting to reference a deleted function
[build] D:\dev\projects\iree\iree\compiler\Dialect\Util\Transforms\FoldGlobals.cpp(360): note: see declaration of 'mlir::iree_compiler::IREE::Util::`anonymous-namespace'::FoldGlobalsPass::FoldGlobalsPass'```